components/datatypes/application.schema.json

```diff
@@ -65,6 +70,71 @@
             "foreground": "Foreground",
             "background": "Background",
             "detached": "Detached"
+          },
+          "xdm:launchType": {
+            "type": "string",
+            "description": "Specifies the launch event type. Install indicates this is the first application launch after the application was installed. Upgrade indicates this is the first application launch after the application was upgraded. Launch is an application launch which is not after an application install or upgrade.",
+            "enum": ["launch", "install", "upgrade"],
+            "meta:enum": {
+              "launch": "Launch",
+              "install": "Install",
+              "upgrade": "Upgrade"
+            }
+          },
+          "xdm:closeType": {
+            "type": "string",
+            "description": "Specifies the close event type. A value of close means the application signaled that it has closed, either by a user action or by the device. Unknown means the application did not signal that it closed, which could indicate an application crash, a system failure, or an improperly configured application.",
+            "enum": ["close", "unknown"],
+            "meta:enum": {
+              "close": "Close",
+              "unknown": "Unknown"
+            }
+          },
```

Comment:
@mjasrotia Can you remove `launchType` and `closeType` as well?  These two fields work with `isLaunch` and `isClose` which you removed.

Reply:
Removed.